__tests__/Component-test.tsx

```diff
@@ -16,3 +19,44 @@ test('flatlist exists when we click on the arrow button', () => {
     expect(queryByTestId('flatlist')).toBeTruthy();
     expect(getByTestId('arrowBtnIcon').props.style).toStrictEqual({resizeMode: 'cover', width: 30, height: 30, transform: [{ rotate: '90deg' }],})
 })
+
+// ToDoComponent Tests
+
+jest.mock('react-native-gesture-handler', () => {})
```

Comment:
do we need this line if we have something similar in line 27

Reply:
removing this leads to test failing